### DISCUSSION TITLE
Discord invite expired

### DISCUSSION THREAD

**Discussion by Tuneezi** [zkSync-Community-Hub/zksync-developers#296] [General]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

Can't seems to join Zksync discord, was shock I'm not in their discord anymore.
Invite link saying expire, how to fix this?

### What is the specific issue or error you're encountering?

Discord invite link expired 

### Can you share the error messages or logs you're receiving, if any?

Discord invite link expired 

### Have you made any recent changes to the contract before encountering this issue?

I have changed my avatar 

### Are there any external libraries or contracts that your contract interacts with?

No

### Can you provide the relevant portions of your contract code where the issue is occurring?

Nill

### Have you tried to isolate the problem, and if so, what were the results?

Same 

### What steps have you already taken to try to resolve the issue?

Contacting friends hete

**Comment by bxpana** [comment]
Please try the following to get access to the zkSync Discord:

- Make sure you have a profile picture in Discord

- Make sure you do not have words such as "admin", "airdrop", etc. in your profile name.
 
- If you were banned, please fill out the form and we'll review as soon as we can https://appeal.zksync.dev/.

If that doesn't work please email us at support@zksync.io
